frontend/src/views/EventDetail.vue

```diff
@@ -17,11 +17,13 @@ export default {
       sortBy: 'popular',
       orderBy: -1,
       questions: [],
-      viewingArchive: false
+      viewingArchive: false,
+      loading: false
     }
   },
   async created() {
     this.setEventId(this.$route.params.eventId)
+    this.showSkeleton()
```

Comment:
you can simply write `this.loading = true` here and it will work :)

Reply:
and then you add a `watch` expression for `event` and whenever an event is loaded, you will simply do `this.loading = false`.